### ISSUE TITLE
Cityscapes datastructure

### ISSUE THREAD

**Issue by S-Beers** [Deci-AI/super-gradients#545]
Hi,
I want to replicate some results, however, I have some issues with loading in the Cityscapes dataset and I can't find anywhere how the dataset structure should be. I saw the list_file and labels_csv_path are needed, but I don't know how to obtain these.

**Comment by Louis-Dupont** [comment]
Hi @S-Beers,
The files that are required in `list_file` and `labels_csv_path` actually need to be downloaded [here](https://deci-pretrained-models.s3.amazonaws.com/cityscape_lists.zip)  and this was not documented, thanks for raising this issue!

The dataset description was just updated in this [PR](https://github.com/Deci-AI/super-gradients/pull/550/files). Here is a copy of what was added, let me know if it is not clear :)

---
    To run this Dataset you need two folders:
    - Cityscape dataset (download here: https://www.cityscapes-dataset.com/downloads/)
        root_dir (in recipe default to /data/cityscapes)
            ├─── gtFine
            │       ├── test
            │       │     ├── berlin
            │       │     │   ├── berlin_000000_000019_gtFine_color.png
            │       │     │   ├── berlin_000000_000019_gtFine_instanceIds.png
            │       │     │   └── ...
            │       │     ├── bielefeld
            │       │     │   └── ...
            │       │     └── ...
            │       ├─── train
            │       │     └── ...
            │       └─── val
            │             └── ...
            └─── leftImg8bit
                    ├── test
                    │     └── ...
                    ├─── train
                    │     └── ...
                    └─── val
                          └── ...
    - Metadata folder (Download here: https://deci-pretrained-models.s3.amazonaws.com/cityscape_lists.zip)
        lists
            ├── labels.csv
            ├── test.lst
            ├── train.lst
            ├── trainval.lst
            └── val.lst
    Example:
        >> CityscapesDataset(root_dir='.../root_dir', list_file='.../lists/train.lst', labels_csv_path='.../labels.csv')

**Comment by shaydeci** [comment]
I am closing this issue down as [dataset setup instructions ](https://github.com/Deci-AI/super-gradients/blob/master/src/super_gradients/training/datasets/Dataset_Setup_Instructions.md ) were added.
